extensions/blocks/recurring-payments/index.js

```diff
@@ -44,9 +44,17 @@ export const settings = {
 		customTextButtonColor: {
 			type: 'string',
 		},
+		align: {
+			type: 'string',
+			default: 'center',
```

Comment:
I've been a bit puzzled by `default:'center'` since having defaults in attributes hides that specific value completely when selected and thus `aligncenter` CSS class never gets applied. I see other blocks do this tho so hence the confusing, but I'm wondering if that is just broken?

Reply:
Removed the default, since it is not really necessary, thanks!